index.html

```diff
@@ -87,48 +88,102 @@ <h3 class="connect">LET’S CONNECT</h3>
           <div class="c1-container">
             <div>
               <h2 class="project">Tonic</h2>
-              <h2 class="project">Tonic</h2>
             </div>
             <div class="descrcontainer">
               <div>
                 <p class="client">CANOPY</p>
-                <p class="client">CANOPY</p>
               </div>
               <div class="mark">
                 <img src="icons/Counter.png" alt="" />
               </div>
               <div>
                 <p class="role">Back End Dev</p>
-                <p class="role">Back End Dev</p>
               </div>
               <div class="mark">
                 <img src="icons/Counter.png" alt="" />
               </div>
               <div>
                 <p class="year">2015</p>
-                <p class="year">2015</p>
               </div>
             </div>
             <div>
               <p class="descrdetail">
                 A daily selection of privately personalized reads; no accounts
                 or sign-ups required.
               </p>
-              <p class="descrdetail">
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required.
-              </p>
             </div>
             <ul class="tagcontainer">
               <li class="tag">html</li>
               <li class="tag">css</li>
               <li class="tag">javaScript</li>
             </ul>
             <div>
-              <button type="button" class="click">See Project</button>
```

Comment:
Please ensure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads) including the technologies. Please do this for all the projects.